.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+<!-- This is a comment. You can remove it and other comments while filling out the template -->
+
+# Summary
+
+<!-- 
+Please provide a **short** summary in a few sentences of what the PR does.
+Try to keep it to as few full sentences as possible.
+-->
+
+## Checklist
+
+<!--
+Check off and cross out points that don't apply:
```

Comment:
Instead of crossing out points that don't apply, most people are not that good with markdown.

I propose to just check the boxes that are applicable for the user.

Reply:
Yes, I was puzzling about this. I updated the template to either just tick off or tick off and add `**N/A**` (for not applicable).